app/bundles/CoreBundle/ErrorHandler/ErrorHandler.php

```diff
@@ -491,18 +491,50 @@ function ($matches) {
                 if (!empty($error['showExceptionMessage'])) {
                     $message = $error['message'];
                 } else {
-                    $message    = 'The site is currently offline due to encountering an error. If the problem persists, please contact the system administrator.';
-                    $submessage = 'System administrators, check server logs for errors.';
+                    unset($error);
+                    $error['message']    = 'The site is currently offline due to encountering an error. If the problem persists, please contact the system administrator.';
+                    $error['submessage'] = 'System administrators, check server logs for errors.';
                 }
-                unset($error);
             }
 
             defined('MAUTIC_OFFLINE') or define('MAUTIC_OFFLINE', 1);
 
             try {
-                ob_start();
-                include __DIR__.'/../../../../offline.php';
-                $content = ob_get_clean();
+                // Get the URLs base path
+                $inDev = false !== strpos($_SERVER['SCRIPT_NAME'], 'index_dev.php');
+                $base  = str_replace(['index.php', 'index_dev.php'], '', $_SERVER['SCRIPT_NAME']);
+
+                // Determine if there is an asset prefix
+                $root = self::$root;
```

Comment:
I can see that this just looks unused but is used in the included paths.php. So this must stay.